src/index.ts

```diff
@@ -293,19 +193,9 @@ const initialize = (apiKey: string, target: Target, options?: Options): Result =
         })
         .then(() => {
           logDebug('Event stream ready', { storage })
-          eventBus.emit(Event.READY, storage)
-
-          if (!hasProxy) {
-            Object.keys(storage).forEach(key => {
-              metrics.push({
-                featureIdentifier: key,
-                featureValue: storage[key],
-                variationIdentifier: evaluations[key]?.identifier || '',
-                count: metricsCollectorEnabled ? 1 : 0,
-                lastAccessed: Date.now()
-              })
-            })
-          }
+          // send only identifiers, for evaluation value we need to use variation method
+          // when we use variation we can track metrics
+          eventBus.emit(Event.READY, Object.keys(storage));
```

Comment:
This is a breaking change, so we'll need a `v2.0.0` version bump. Do we want to do this?